vtex-apps/tiktok-tbp
# Feature/brpa 711 tiktok sdk

#### What is the purpose of this pull request?
- Create a `useTikTokSdk` react hook to load the required scripts.

#### What problem is this solving?
- Load the TikTok SDK when opening the admin page.

#### How to test it?
Open [this workspace](https://tiktok--sandboxbrdevbrl.myvtex.com/admin/tiktok) and then the Network tab in your browser. When the page is loading, it should load this script: `tbp_external_platform-v2.1`.

#### Screenshots or example usage
<img width="1440" alt="Screen Shot 2022-01-19 at 11 41 14" src="https://user-images.githubusercontent.com/1629129/150154672-097c0dce-d9eb-4319-8912-02e52da7e5ac.png">

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/109fP7pua6Osgw/giphy.gif)

#### Types of changes
